beacon_node/src/cli.rs

```diff
@@ -622,6 +623,14 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                        [disabled by default].")
                 .requires("slasher")
         )
+        .arg(
+            Arg::with_name("slasher-backend")
+                .long("slasher-backend")
+                .help("Set the database backend to be used by the slasher.")
+                .takes_value(true)
+                .possible_values(slasher::DatabaseBackend::VARIANTS)
```

Comment:
This is cool!

Reply:
since I worked this out I've been putting it in all my PRs haha